hbase_storage/src/main/java/com/splicemachine/hbase/ZkUtils.java

```diff
@@ -315,10 +317,17 @@ public static long nextSequenceId(String counterNode) throws IOException{
 		 */
 
         RecoverableZooKeeper rzk=ZkUtils.getRecoverableZooKeeper();
-        int maxTries=10;
+        int maxTries=Integer.MAX_VALUE;
         int tries=0;
         while(tries<=maxTries){
             tries++;
+            if (tries > 1) {
+                try {
+                    Thread.sleep(50);
```

Comment:
If we are sleeping, can you use `PipelineUtils.getPauseTime()` or something similar to do an exponential backoff and add jitter?